app/models/blockchain/block.rb

```diff
@@ -21,4 +21,24 @@
 #
```

Comment:
The section about db structure should be removed from this file, since there's no such table as "blockchain_blocks"
Same for blockchain_slots and blockchain_transactions